### DISCUSSION TITLE
Problem with upscaling. Hours for it.

### DISCUSSION THREAD

**Discussion by Foressee** [lllyasviel/Fooocus#3515] [Q&A]
Hi everyone. I hope someone knew what to do with this problem.
I have problem with upscaling, its take an hour maybe for one picture.
Its stopes like this:
![2024-08-14_01-26-24](https://github.com/user-attachments/assets/06f468d3-541c-4bd2-878f-79162fb9131d)
And only after hour or so, its start generate.
With ordinary generation of arts, its do fast work but not with this.
![2024-08-14_01-34-30](https://github.com/user-attachments/assets/ecd6011b-9932-4d8b-8077-b4f089856370)

**Comment by Rogala** [comment]
It all depends on the power of your PC, and Performance mode, the better it is the longer it takes, if you have a slow processor, little RAM and HDD disk, the initial calculation of the task takes a long time, then it is all transferred to the video card, and now its parameters play a role, the GPU and the amount of video memory, the less it is the more often the task is taken from swap, the weaker the GPU the more it processes each step.

**Reply by Foressee** [reply]
yeah, but problem that I have good PC and generation goes very fast. Problem only with upscale

**Reply by Rogala** [reply]
Do not use Animagine XL V3.1 check just with Base model animaPencilXL_v500

**Reply by Foressee** [reply]
same, already try it

**Reply by Rogala** [reply]
And do you have the upscaler file itself?
`Fooocus\models\upscale_models\fooocus_upscaler_s409985e5.bin`

**Reply by Rogala** [reply]
These lines are obviously some kind of error, but only the developers can explain it
```
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
left over keys: dict_keys(['cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'])
```

**Reply by Foressee** [reply]
yeah. I check it and I have it

**Reply by Foressee** [reply]
extra {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_l.text_projection'}
left over keys: dict_keys(['cond_stage_model.clip_l.transformer.text_model.embeddings.position_ids'])

this ok I think, I combine 2 models when generate arts
